libsplinter/src/node_id/store/diesel/operations/set_node_id.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2018-2021 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::super::{NodeID, NodeIdStoreError};
```

Comment:
Current convention is to limit super calls to single depth.

Also, order your statements as:

1. std lib
2. extern crates
3. `crate::` uses
4. `super::` uses
5. `self::` uses

All separated by a whitespace